pkg/synopsysctl/cmd_root.go

```diff
@@ -70,7 +73,10 @@ var rootCmd = &cobra.Command{
 
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute() {
+func Execute(version string) {
+	rootCmd.Version = version
+	synopsysOperatorImage = fmt.Sprintf("gcr.io/saas-hub-stg/blackducksoftware/synopsys-operator:%s", version)
```

Comment:
version should be master

Reply:
@msenmurugan I believe this concern is resolved by changes from #502 and #504.  Do you agree?